compiler/plugins/target/ROCM/builtins/tuning/test/spec_gfx942.mlir

```diff
@@ -54,3 +54,58 @@ hal.executable public @main {
     }
   }
 }
+
+// -----
+
+// CHECK-LABEL:  func.func @attention_2x10x4096x64x64x64_f16
+// CHECK:          iree_linalg_ext.attention
+// CHECK-SAME:       __tuning_spec_applied__
+
+// MI300X-LABEL:  func.func @attention_2x10x4096x64x64x64_f16
+// MI300X:          iree_linalg_ext.attention
+// MI300X-SAME:       __tuning_spec_applied__
+
+hal.executable public @main {
+  hal.executable.variant public @rocm_hsaco_fb target(<"rocm", "rocm-hsaco-fb">) {
+    hal.executable.export public @attention ordinal(0) layout(#hal.pipeline.layout<constants = 2, bindings = [#hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">, #hal.pipeline.binding<storage_buffer, Indirect>], flags = Indirect>) {
+    ^bb0(%arg0: !hal.device):
+      %x, %y, %z = flow.dispatch.workgroup_count_from_slice
+      hal.return %x, %y, %z : index, index, index
+    }
+    builtin.module {
+      // expected-remark@+1 {{Applied transform configuration strategy @iree_default_tuning_spec_gfx942::@__kernel_config}}
+      func.func @attention_2x10x4096x64x64x64_f16() {
```

Comment:
I think we should also have at least one negative test where the attention config is expected *not to apply*